backup_and_restore/replacing-failed-master.adoc

```diff
@@ -18,10 +18,15 @@ To replace a single master host:
 
 * xref:../backup_and_restore/replacing-failed-master.adoc#restore-remove-failed-master_replacing-failed-master-host[Remove the member from the etcd cluster].
 
+* If the etcd certificates for the master host have expired, xref:../backup_and_restore/replacing-failed-master.adoc#restore-regenerate-etcd-certs_replacing-failed-master-host[regenerate etcd certificates].
+
```

Comment:
Lines 21 and 23 should be switched:
If the etcd certificates for the master host are valid, xref:../backup_and_restore/replacing-failed-master.adoc#restore-add-master_replacing-failed-master-host[Add the member back to the etcd cluster].

If the node is new without any certificates or certificates are no longer valid, xref:../backup_and_restore/replacing-failed-master.adoc#restore-regenerate-etcd-certs_replacing-failed-master-host[regenerate etcd certificates and add/recover the member back to the etcd cluster].